.github/workflows/RedeployInferenceACI.yml

```
@@ -0,0 +1,74 @@
+name: GitHub Actions Deploy AI4Orca ML ACI
+on:
+  # Triggers the workflow on merges only
+  push:
+    branches: [ main ]
+    paths:
+      - "InferenceSystem/configProduction/**"
+
+  # Allows you to run this workflow manually from the Actions tab
```

```suggestion
  # Allows you to run this workflow manually
```
Not saying it's not true but I think this sentence is a bit misleading, unaware person might think that triggering from GUI is the only option but in reality you can send POST request as well.